code/game/objects/items/cards_ids.dm

```diff
@@ -767,11 +787,7 @@
 	if(isnull(account))
 		to_chat(user, span_warning("The account ID number provided is invalid."))
 		return FALSE
-	if(old_account)
-		old_account.bank_cards -= src
-		account.account_balance += old_account.account_balance
-	account.bank_cards += src
-	registered_account = account
```

Comment:
I'm unsure what this was doing, without looking deeper it implies that we're generating money out of thin air (the old account's balance never depletes) but that's assuming the old account can still be accessed